scala/zio-http/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.4.9
```

Comment:
is this file require @whiplash ? I think it should be added in `.gitignore` file

Reply:
We've talked about it. I'd keep it, but we can work without it.

Reply:
Yeah, I prefer removing it, but it could be done in an other `PR` (in order to remove for all scala based frameworks)